evergreen.yml

```diff
@@ -465,7 +465,7 @@ functions:
           docker run -i --platform="linux/amd64" --rm -v "$PWD":/pwd \
           --env-file silkbomb.env \
           artifactory.corp.mongodb.com/release-tools-container-registry-public-local/silkbomb:1.0 \
-          upload --silk-asset-group ${SILK_ASSET_GROUP} --sbom-in /pwd/$SBOM_FINAL
+          upload --silk-asset-group ${SILK_ASSET_GROUP} --sbom-in /pwd/$SBOM_FINAL --force
```

Comment:
Silkbomb now fails if the SBOM is not in CycloneDX 1.5 format. Our is using the latest CycloneDX spec, 1.6.
We are manually checking for vulnerabilities anyway, so forcing the upload is Ok.
There is [DEVPROD-11508](https://jira.mongodb.org/browse/DEVPROD-11508) to track supporting the new spec version.